operator-metadata/automation/tests/test_bundle_automation.py

```diff
@@ -82,11 +83,11 @@ def test_update_new_csv_format(self):
             csv_file_path=self.NEW_CSV_INTERNAL_PULL_SPECS_FILE_PATH,
             bundle_versions=["2.4.0-0", "2.5.0-0"],
             replacement_map={
-                "strimzi-rhel9-operator:2.5.0-1": self.OPERATOR_PULL_SPEC_REPLACEMENT,
-                "kafka-34-rhel9:2.5.0-1": self.KAFKA_PREVIOUS_PULL_SPEC_REPLACEMENT,
-                "kafka-35-rhel9:2.5.0-1": self.KAFKA_CURRENT_PULL_SPEC_REPLACEMENT,
-                "bridge-rhel9:2.5.0-1": self.BRIDGE_PULL_SPEC_REPLACEMENT,
-                "maven-builder-rhel9:2.5.0-1": self.MAVEN_PULL_SPEC_REPLACEMENT
+                "strimzi-rhel8-operator:2.5.0-1": self.OPERATOR_PULL_SPEC_REPLACEMENT,
```

Comment:
What's the fundamental difference between the two scenarios exercised by this test?

Reply:
Only difference is that the automation will work the same whether the CSV referenced internal pull specs or external pull_specs

Reply:
Thanks.  Maybe some comments would help just to capture the intent.